gulp.d/tasks/build.js

```diff
@@ -47,7 +47,8 @@ module.exports = (src, dest, preview) => () => {
         },
       },
     ]),
-    postcssVar({ preserve: preview }),
+    // NOTE because css vars are our theme switching mechanism we must preserve them with preserve: true
+    postcssVar({ preserve: true }),
```

Comment:
The result of this is I think about 150 extra lines of css, nothing major. By preserving the css vars it leaves them as is instead of trying to resolve the values into plain css